packages/contracts-bedrock/contracts/universal/ProxyAdmin.sol

```diff
@@ -6,25 +6,40 @@ import { Owned } from "@rari-capital/solmate/src/auth/Owned.sol";
 import { Lib_AddressManager } from "../legacy/Lib_AddressManager.sol";
 import { L1ChugSplashProxy } from "../legacy/L1ChugSplashProxy.sol";
 
+// Define static interfaces of these proxies so that we can easily
+// use staticcall on the getters we need.
+interface IStatic_ERC1967Proxy {
+    function implementation() external view returns (address);
+
+    function admin() external view returns (address);
+}
+
+interface IStatic_L1ChugSplashProxy {
+    function getImplementation() external view returns (address);
+
+    function getOwner() external view returns (address);
+}
+
 /**
  * @title ProxyAdmin
- * @dev This is an auxiliary contract meant to be assigned as the admin of a Proxy, based on
- *      the OpenZeppelin implementation. It has backwards compatibility logic to work with the
- *      various types of proxies that have been deployed by Optimism.
+ * @dev This is an auxiliary contract meant to be assigned as the admin of an ERC1967 Proxy,
+ *      based on the OpenZeppelin implementation. It has backwards compatibility logic to work with
+ *      the various types of proxies that have been deployed by Optimism.
  */
 contract ProxyAdmin is Owned {
     /**
      * @notice The proxy types that the ProxyAdmin can manage.
      *
-     * @custom:value OpenZeppelin     Represents the OpenZeppelin style transparent proxy
+     * @custom:value ERC1967          Represents the ERC1967 transparent proxy
      *                                interface, this is the standard.
      * @custom:value Chugsplash       Represents the Chugsplash proxy interface,
      *                                this is legacy.
      * @custom:value ResolvedDelegate Represents the ResolvedDelegate proxy
      *                                interface, this is legacy.
      */
     enum ProxyType {
-        OpenZeppelin,
+        Unknown,
```

Comment:
Agreed with this, we should remove the unknown type

Reply:
I realized that was the intent, but still it seems like pretty subtle behavior to me, and not that difficult to call `setProxyType`. 
Not having a default type is also more future proof IMO, especially in case we (god forbid), adopt yet another proxy type in the future.

Reply:
OK.